stdlib/public/core/StringInterpolation.swift

```diff
@@ -197,6 +197,128 @@ public struct DefaultStringInterpolation: StringInterpolationProtocol, Sendable
   }
 }
 
+extension DefaultStringInterpolation {
+  /// Interpolates the given optional value's textual representation, or the
+  /// specified default string, into the string literal being created.
+  ///
+  /// You don't need to call this method directly. It is used by the compiler
```

Comment:
Here and in the other places this phrasing appears, missing a contraction:

```suggestion
  /// You don't need to call this method directly. It's used by the compiler
```